docs/config.md

```diff
@@ -145,7 +145,8 @@ environment variables for the OTLP exporter:
 
 > ⚠️ **Do NOT use in production.**
 >
-> Prometheus exporter uses [an unreliable, unmaintained legacy .NET component](https://github.com/dotnet/runtime/issues/28658#issuecomment-462062760).
+> Prometheus exporter [is intended for dev inner-loop,
+> there is no plan to make it production ready](https://github.com/open-telemetry/opentelemetry-dotnet/blob/622583c6ffaea2e3d83d4fcb8935c6a8dbc6f81f/src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md?plain=1#L10-L16).
```

Comment:
What is the `dev inner-loop` and why are we featuring this exporter here and not in a dev doc if it's not for production?

Reply:
I think the wording is not perfect. That is why I suggested to change it in my review

Reference: https://www.getambassador.io/docs/telepresence/latest/concepts/devloop/#what-is-the-inner-dev-loop

Reply:
Taken from OTel SDK side. I think it is useful for developing/debugging scnearios.

Reply:
> Why are we featuring this exporter here and not in a dev doc if it's not for production 

Good point. Personally, I think it may still be usable... But we may consider removing it completely. I think we considered it during the SIG meeting, but first we agreed to just mark it as "non-production".